File: data/readable_format/fortran/file2220.f90
module greet_mod
contains
    subroutine greet_world()
        write(*, '(a)') 'Greet World'
    end subroutine greet_world
end module greet_mod

program test_greet
    use greet_mod
    call greet_world()
end program test_greet

File: data/readable_format/cpp/file2220.cpp
#include <iostream>

class GreetMod {
public:
    void greet_world() const {
        std::cout << "Greet World" << std::endl;
    }
};

int main() {
    GreetMod gm;
    gm.greet_world();
    return 0;
}